.circleci/config.yml

```diff
@@ -277,6 +277,10 @@ jobs:
           name: Run e2e file upload features test
           command: |
             python -m scripts.run_e2e_tests --skip-install --suite="fileUploadFeatures"
+      - run:
+          name: Run e2e play translations tesst
+          command: |
+            python -m scripts.run_e2e_tests --skip-install --suite="playTranslations"
```

Comment:
Move this to the `e2e_classroom_page` section, since we want to keep this as a top level test.

Reply:
done!